crates/daphne-server/src/router/mod.rs

```diff
@@ -160,9 +160,9 @@ impl AxumDapResponse {
     }
 
     pub fn new_error<E: Into<DapError>>(error: E, metrics: &dyn DaphneServiceMetrics) -> Self {
-        // trigger abort if transition failures reach this point.
+        // trigger abort if report errors reach this point.
```

Comment:
Might as well fix grammar here.
```suggestion
        // Trigger abort if report errors reach this point.
```